analytics/analytics/models.py

```diff
@@ -8,12 +58,21 @@ class Job(models.Model):
     project_id = models.PositiveBigIntegerField()
     name = models.CharField(max_length=128)
     started_at = models.DateTimeField()
-    duration = models.FloatField()
+    duration = models.DurationField()
     ref = models.CharField(max_length=256)
     tags = ArrayField(base_field=models.CharField(max_length=32), default=list)
     package_name = models.CharField(max_length=128)
 
-    # Fields allow null to accomodate historical data
+    # Whether this job ran in the cluster or not
+    aws = models.BooleanField(default=True)
```

Comment:
Is there a reason to model this as `aws` instead of a choices field for `aws/uo` and others in the future?

Reply:
Also, I think the comment could be migrated to `help_text` in the field.

Reply:
The main reason is that if `aws=True` (the job ran in the cluster), then we have extra data for it. If not, we don't have that data. The rest doesn't _really_ matter at the moment, since the data we're gathering is mainly focused on the things we control and can optimize (jobs/runners in the cluster). 

Where exactly a runner ran outside the cluster also might not be easy to determine. For example, in the case of UO runners, you could just look at the runner name/description to see if it starts with UO. But sometimes you can't access the runner, and that might not generalize to other "classes" of runner.

If in the future we have a concrete way to determine where a runner ran, I'd be happy to make the change.